src/main/java/com/minecolonies/coremod/colony/buildings/BuildingBuilder.java

```diff
@@ -246,84 +360,91 @@ public void resetNeededResources()
     }
 
     /**
```

Comment:
Information about a resource. (He wants a period after the first sentence in each javadoc.)